tests/vm/tnimnode.nim

```diff
@@ -0,0 +1,69 @@
+import macros
```

Comment:
This needs a "discard" spec section. Look at other tests to see how they look like.